app.go

```diff
@@ -125,6 +111,10 @@ func (og optionGroup) String() string {
 }
 
 // StartTimeout changes the application's start timeout.
+// This controls the total time that all OnStart hooks have to complete.
```

Comment:
I think linking to this here would provide some value.
```suggestion
// This controls the total time that all [OnStart] hooks have to complete.
```